src/main/frontend/extensions/pdf/pdf.css

```
@@ -56,15 +56,22 @@ input::-webkit-inner-spin-button {
     position: relative;
   }
 
-  &-toolbar {
+&-header {
+    display: flex;
+    justify-content: flex-end;
     position: absolute;
     top: 0;
-    right: 20px;
-    width: 90%;
-    height: 42px;
+    right: 0px;
+    width: 100%;
+    padding: 0 20px;
```

```suggestion
    padding: 8px 20px;
```
Keeping the top/bottom padding on the header would allow us to drag the window by clicking around the toolbar.